pkg/osquery/tables/wifi_networks/internal/assets/get-networks.ps1

```diff
@@ -0,0 +1,34 @@
+function Get-Networks {
+    function Convert-ByteArrayToString {
+        [CmdletBinding()] Param (
+            [Parameter(Mandatory = $True, ValueFromPipeline = $True)] [System.Byte[]] $ByteArray
+            )
+
+        $Encoding  = New-Object System.Text.ASCIIEncoding
+        $Encoding.GetString($ByteArray)
+    }
+
+    Add-Type -Path ".\nativewificode.cs"
+    $WlanClient = New-Object NativeWifi.WlanClient
+
+    $WlanClient.Interfaces | ForEach-Object { $_.Scan() }
+
+    #check scan progress for each interface every 50 milliseconds
+    $scanInProgress = "false"
+    do {
+      $scanInProgress = "false"
+      $WlanClient.Interfaces | ForEach-Object {
+        $ip = $_.scanInProgress
+        if ($ip -eq "True") {
+          $scanInProgress = "true"
+        }
+      };
+      Start-Sleep -Milliseconds 100
+    } while ($scanInProgress -eq "true")
```

Comment:
I can't really parse this. Will this ever timeout? I guess the go code will handle that

Reply:
Yea, the go code handles it. It doesn't time out, but you're probably right that it should.

Reply:
Go seems good enough. I don't want to get too fancy in our non-core languages